### DISCUSSION TITLE
[Nodriver] How to interact with elements in shadowroot?

### DISCUSSION THREAD

**Discussion by dbhmw** [ultrafunkamsterdam/undetected-chromedriver#1942] [Q&A] [answered]
After the cloudflare update the iframe is in the closed shadowRoot. How to navigate and interact with elements in there as the select function throws an error?

**Comment by lol157** [comment]
@dbhmw Have you found a solution?

**Reply by dbhmw** [reply]
no

**Comment by devblack** [comment, marked as answer]
The solution is here #1998
